test/precompiled/precompiled_test.h

```diff
@@ -2,7 +2,5 @@
 
 #include "precompiled/precompiled.h"
 
-#include "test/test_common/printers.h"
```

Comment:
This is required to have pretty printing work correctly. It will compile but won't actually pretty print without it. Any reason to remove?